packages/batchclient/src/index.ts

```diff
@@ -0,0 +1,127 @@
+import { formatsByCoinType } from '@ensdomains/address-encoder';
```

Comment:
Can't all of this be replaced by a simple call to Ethers to resolve the name now?